tasks/zadanie_8.php

```diff
@@ -1,18 +1,10 @@
 <?php
 
-/*
-
-Napisz kod który użyje zmiennej 
-    - $role (oznaczająca rolę w systemie) 
-    - $isLogged (oznaczająca czy jest zalogowany)
-Jeśli rola jest równa "admin" oraz jest zalogowany to wyświetl na ekranie "Dostęp przyznany"
-W każdym innym przypadku wyświetl komunikat "Brak dostępu"
-
-*/
-
 $role = $params[0]; // tej linijki nie ruszamy :)
 $isLogged = $params[1]; // tej linijki nie ruszamy :)
 
-// ...
-
-echo "Dostęp przyznany";
\ No newline at end of file
+if ($role === "admin" && ($isLogged === "true" || $isLogged === true)) {
```

Comment:
Tutaj tylko $isLogged przyjmijmy że jest typu bool :D 
```suggestion
if ($role === "admin" && $isLogged === true) {
```